### PR TITLE
Feat implemented touppercase

### PR DESCRIPTION
**Pull request by RameshSuthar**

### CODE REVIEW

File: implementations/lowerToUpperCase.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+String.prototype.toUpperCaseString = function (str){
+    let upperCaseString = ""
+    for(let i=0;i<this.length;i++){
+        upperCaseString += String.fromCharCode(this.charCodeAt(i) - 32);
+    }
+    return upperCaseString
+}
+
+console.log('hacktober fest is a one month festival of open source software'.toUpperCaseString());
```
Comment (ashokdey) [inline implementations/lowerToUpperCase.js]: **no-console** in final code

File: implementations/lowerToUpperCase.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * 
+ *  */
+
+String.prototype.toUpperCaseString = function (str){
+    let upperCaseString = ""
+    for(let i=0;i<this.length;i++){
+        upperCaseString += String.fromCharCode(this.charCodeAt(i) - 32);
+    }
+    return upperCaseString
+}
```
Comment (ashokdey) [inline implementations/lowerToUpperCase.js]: Invalid implementation. Will not work for:

```javascript
const a = '@123 hello - world';
```

File: implementations/lowerToUpperCase.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ *        To Implement this function I have used charCodeAt() to convert the character into it's ASCII value
+ *        and then I subtracted 32 from that ASCII value to convert that ASCII value of lower letter to the ASCII
+ *        value of Upper letter.
+ *        Finally I used String.fromCharCode() to convert the ASCII value into character.
+ * 
+ * 
+ *  */
+
+String.prototype.toUpperCaseString = function (str){
```
Comment (ashokdey) [inline implementations/lowerToUpperCase.js]: **Invalid argument**

File: implementations/lowerToUpperCase.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    let upperCaseString = ""
+    for(let i=0;i<this.length;i++){
+        upperCaseString += String.fromCharCode(this.charCodeAt(i) - 32);
```
Comment (ashokdey) [inline implementations/lowerToUpperCase.js]: `str` as an argument and using `this`?

**Comment by ashokdey** [comment]
Closing this because #11 has already fixed it and you made a duplicate PR. Considering your dedication, I can assign you an issue which only you can work on. So just wait for a new issue.

You need more practice. All the best.

Thanks

**Comment by RameshSuthar** [comment]
Dear Sir,

I solved that problem can I make a pull request now?

Thank you
Ramesh Suthar.

**Comment by ashokdey** [comment]
@RameshSuthar you can keep the solution for your reference and wait for other issues. This issue is going to close now.

**Comment by RameshSuthar** [comment]
can I implement  String.prototype.toLowerCase() method?

**Pull request closed without merging**